tests/e2e/instrumentation-dotnet-multicontainer/01-assert.yaml

```diff
@@ -18,15 +18,15 @@ spec:
     - name: CORECLR_PROFILER
       value: "{918728DD-259F-4A6A-AC2B-B85E1B658318}"
     - name: CORECLR_PROFILER_PATH
-      value: /otel-auto-instrumentation/linux-x64/OpenTelemetry.AutoInstrumentation.Native.so
+      value: /otel-auto-instrumentation-dotnet/linux-x64/OpenTelemetry.AutoInstrumentation.Native.so
```

Comment:
I think we should try to implement this functionality without modifying the existing tests.  The new tests should set the feature flag and these tests should not.  I am pretty sure our current test setup is too inflexible to handle that, so we might need changes in another PR, but it would be nice.

Reply:
I'm not sure if splitting this variables using featureflag will be valuable. Another few thousands of lines of the same tests, highly increased build time. I would like to leave it as it is now, and if there will be a need I would do that in separated PR.

Reply:
Looking at the PR again I realize now that the things that are changing aren't hidden behind the feature gate.  While _technically_ breaking changes, these are all env vars/paths that the operator is injecting/using and the user should not be depending on.  I agree that it is ok to leave these changes outside of the featuregate.